polymer.externs.js

```diff
@@ -426,8 +426,10 @@ PolymerElement.prototype.importHref = function(href, onload, onerror) {};
 
 /**
  * Delete an element from an array.
- * @param {!Array} array
- * @param {*} item
+ * @param {!Array|string} array Path torray from which to remove the item (or
```

Comment:
good catch! Fixed. I copied this directly from http://polymer.github.io/polymer/ so it should probably be fixed there too :-).

CC: @arthurevans